helpers/questions_filter/questions_filter.py

```diff
@@ -0,0 +1,33 @@
+from dateutil import parser
+from dateutil.relativedelta import relativedelta
+from graphql import GraphQLError
+
+
+def filter_questions_by_date_range(query, start_date, end_date):
```

Comment:
Kindly change query to questions, same to line 14

Reply:
@Bonifase This has been changed.